CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ### NEXT
 
+- AV1: Add support for DD extension header forwarding ([#1610](https://github.com/versatica/mediasoup/pull/1610))
```

Comment:
You have started a dark battle against dots at the end of sentences and I won't give up.


```suggestion
- AV1: Add support for DD extension header forwarding ([#1610](https://github.com/versatica/mediasoup/pull/1610)).
```